.rubocop.yml

```diff
@@ -3,6 +3,7 @@ AllCops:
   UseCache: false
   Exclude:
    - 'projects/**/*'
+   - 'vendor/**/*'
```

Comment:
@gkman that's what you were missing. How do I know? I made `.travis.yml` like so at some point
```yaml
script:
  - pwd
  - ls -la
```
which revealed the `vendor` directory Travis CI creates in the workspace directory. Rubocop was failing on files in that directory. Hence the fix was to exclude `vendor`

Reply:
got it